### PR TITLE
ea_commands: add policy_authorize_nv

### PR DESCRIPTION
**Pull request by gomesj**
Although binding is already present, policy_authorize_nv was missing from the supported enhanced authorization commands.
Add a simple implementation to fill the gap.

### CODE REVIEW

**Comment by Superhepper** [comment]
Thank you for the PR. Could you please also add some tests for this Context method?

**Comment by gomesj** [comment]
Good point, added a simple test case in my last push.

File: tss-esapi/tests/integration_tests/common/mod.rs [hunk 1]
```diff
@@ -432,3 +434,45 @@ pub fn create_public_sealed_object() -> Public {
         .build()
         .expect("Failed to create public structure.")
 }
+
+#[allow(dead_code)]
```
Comment (wiktor-k) [inline tss-esapi/tests/integration_tests/common/mod.rs:438]: Is this really necessary?

**Review by wiktor-k — APPROVED** [review]
Looks good to me although I think @Superhepper meant a doctest in the Context function you added. (Note that they are slightly harder to write but you can take a look at other functions for inspiration).

**Comment by Superhepper** [comment]
Well actually I meant both. The methods need to be documented a little better especially when it is a Context method. Unfortunately not all Context methods are documented properly at the moment. But there are a few good examples that you can have a look at (e.g. ```ecdh_key_gen```).

File: tss-esapi/tests/integration_tests/common/mod.rs (hunk 1 above)
Comment (gomesj) [reply tss-esapi/tests/integration_tests/common/mod.rs]: I originally followed the example of most other functions in this file, but apparently it was not.

**Review by Superhepper — APPROVED** [review]

**Pull request merged**